assets/cluster-monitoring-operator/prometheus-rule.yaml

```
@@ -409,6 +409,21 @@ spec:
       labels:
         workload_type: deploymentconfig
       record: namespace_workload_pod:kube_pod_owner:relabel
+  - name: openshift-etcd-telemetry.rules
+    rules:
+    - expr: sum by (instance) (etcd_mvcc_db_total_size_in_bytes{job="etcd"})
+      record: cluster:etcd_mvcc_db_total_size_in_bytes:sum
+    - expr: histogram_quantile(0.99, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket{job="etcd"}[5m]))
+        by (instance, le))
```

(suggestion) I find it more readable when the `by()` clause is right after `sum`.

```suggestion
    - expr: histogram_quantile(0.99, sum by (instance, le) (rate(etcd_disk_wal_fsync_duration_seconds_bucket{job="etcd"}[5m]))
       )
```